valmlord/gulpify
# Feature/webp convertation

## Description
This pull request introduces new tasks to duplicate all existing images in the project to the WebP format. It utilizes the `gulp-webp` plugin for efficient image conversion.

## Changes
### Commit 1: Add tasks for duplicating images to WebP
- Added `gulp-webp` package to `package.json`.
- Created new Gulp tasks to convert existing images (e.g., PNG, JPEG) to WebP format.
- Configured the tasks to output the WebP images alongside the original files.

## How to Test
1. Place image files (e.g., PNG, JPEG) in the appropriate directory (e.g., `app/assets/img`).
2. Execute the relevant Gulp task (e.g., `gulp compressIMG`) to convert the images to WebP.
3. Verify that the WebP images are correctly generated alongside the original files.


## Additional Notes
- This update aims to improve performance by providing WebP versions of images for better web optimization.
